Calling/NodeApi/index.ts

```diff
@@ -56,8 +54,8 @@ const [
 });
 
 const blobContainerName = 'files';
-const fileMetadataTableName = 'fileMetadata';
-const userDetailTableName = 'userDetails';
+const tableName = 'fileMetadata';
```

Comment:
Might need to pull? These names were made more specific since we're adding more tables

Reply:
fixed